CHANGES.rst

```diff
@@ -8,6 +8,10 @@
   - ``flycheck-go-build-install-deps`` turns on dependency installation for ``go test``
     as well as ``go build`` [GH-1003]
 
+- New syntax checkers:
+
+  - Elixir with ``dogma`` [GH-969]
```

Comment:
I'm sorry to bother you again, but as a convention we list new syntax checkers first, as they are likely the most interesting change for users.

If you rebase onto master again, you'll see another entry for a new syntax checker.  Please add yours to it.